rs/index/src/multi_spann/builder.rs

```diff
@@ -37,6 +38,17 @@ impl MultiSpannBuilder {
         Ok(())
     }
 
+    pub fn invalidate(&mut self, user_id: u128, doc_id: u128) -> Result<bool> {
```

Comment:
```suggestion
    pub fn invalidate(&self, user_id: u128, doc_id: u128) -> Result<bool> {
```
Don't need `mut` here